standup/auth0/models.py

```diff
@@ -3,11 +3,11 @@
 
 
 class IdToken(models.Model):
-    """Relates an ID token to a user
+    """Associates a user with their id token
 
-    These tokens need to be renewed on a periodic basis and if they fail, they cause the Django
-    session to expire.
+    These tokens need to be renewed on a periodic basis and if they fail, they force Django logout.
 
     """
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
-    id_token = models.TextField(max_length=100, null=True)
+    id_token = models.TextField(help_text='The id token for this user.')
```

Comment:
I assume we'll have to lookup entries based on this field though, and some DB engines have trouble with indexing a text blob field. I think we should have this be a `CharField` with a large max_length, maybe 200, and set `db_index=True`?